e2e/katz_online_test/test_online_call_get_on_inconsistent/test.yml

```diff
@@ -0,0 +1,29 @@
+- query: >-
+    CALL katz_centrality_online.get() YIELD node, rank
+    RETURN node.id AS node_id
+    ORDER BY rank DESC, node_id ASC;
+  output:
+      - node_id: 1
+      - node_id: 4
+      - node_id: 0
+      - node_id: 3
+      - node_id: 5
+      - node_id: 6
+      - node_id: 2
+
+- query: >-
+    CALL katz_centrality_online.get() YIELD node, rank
+    RETURN node.id AS node_id
+    ORDER BY rank DESC, node_id ASC;
+  output:
+      - node_id: 1
+      - node_id: 4
+      - node_id: 0
+      - node_id: 3
+      - node_id: 5
+      - node_id: 2
+
+- query: >-
+    CALL katz_centrality_online.get() YIELD *;
+  exception: >-
+    Graph should be inconsistent at this point!
```

Comment:
Nothing major, but it would be good if us across MAGE were consistent in using the same error message for this case.
So far, I've been using `"Graph has been modified and is thus inconsistent with cached {NAME} scores; to update them, please call set/reset!"`

Reply:
I agree, but the emphasis on consistency part should be in algorithm's code rather than in tests. Testing module is implemented just to check whether there is an exception. The comment below it does nothing, but it's useful to check what is really going on.

Reply:
Thanks!